stories/responsive-table/index.jsx

```diff
@@ -196,6 +195,7 @@ export default decorator('ResponsiveTable', TableDocs, TableReadme).add('Respons
     direction: 'asc',
   };
   const [sort, setSort] = React.useState(defaultSort);
+  const [loading, setLoading] = React.useState(false);
   const handleSort = (sortName, direction) => {
     console.log(`Column ${sortName} is now sorted in ${direction === 'desc' ? 'asc' : 'desc'} direction`);
```

Comment:
maybe add if __DEV__ ?

Reply:
This log is in story, it won't be in a package